src/altair/state_transition/helpers.rs

```diff
@@ -117,7 +117,12 @@ pub fn compute_proposer_index<
     loop {
         let shuffled_index = compute_shuffled_index((i % total) as usize, total, seed, context)?;
         let candidate_index = indices[shuffled_index];
-        let i_bytes: [u8; 8] = (i / 32).to_le_bytes();
+        #[cfg(not(feature = "serde"))]
+        let i_bytes: [u8; 4] = (i.saturating_div(32)).to_le_bytes();
+
+        #[cfg(feature = "serde")]
+        let i_bytes: [u8; 8] = (i.saturating_div(32)).to_le_bytes();
+
```

Comment:
what is happening here?

Reply:
So, I am trying to make this repo compile to target --target=wasm32-unknown-unknown
I get this error when i try to compile this section of code to that target

```
 let i_bytes: [u8; 8] = (i/32).to_le_bytes();
    |                      -------   ^^^^^^^^^^^^^^^^^^^^ expected an array with a fixed size of 8 elements, found one with 4 elements
    |                      |
    |                      expected due to this

```